### ISSUE TITLE
Contibutor call: 9 January 2021

### ISSUE THREAD

**Issue by joostdecock** [freesewing/freesewing#766]
You are all invited to the contributor call.
Please leave a comment to put things on the agenda. This issue will be updated after the call with notes and todos.

## About

 - Location: https://meet.freesewing.org (Zoom call)
 - Date: Saturday 9 January, 2021 17:00 UTC, which is:
   - Los Angeles: 09:00
   - Chicago: 11:00
   - New York: 12:00
   - São Paulo: 14:00
   - London: 17:00
   - Paris: 18:00
   - Johannesburg: 18:00
   - Moscow: 20:00
   - Mumbai: 22:30
   - Hong Kong: 01:00 on Sunday
   - Tokyo: 02:00 on Sunday
   - Sydney: 04:00 on Sunday

## Agenda

 - Happy New Year :fireworks: :slightly_smiling_face: 
 - Todos from last call: (aka Fantastic deadlines and how to miss them)
   - Setup newsletter platform (see #685)
   - Add newsletter sign-up form to the website (see #720)
   - Draft first edition of the newsletter (see #721)
   - Renames all `master` branches to `main` (see #722)
 - v2.11 milestone
 - Fit issues in Paco (#770)
 - 2020 roundup
 - Discussion

## Notes

 - Before the formal start of the call, we discussed [the pull request](https://github.com/freesewing/markdown/pull/146) by @tangerineshark with @Leo-Luuk and agreed that we'd add a specific FAQ to the site and link to that in the getting started guide rather than try to explain everything there
 - This was the first call of the new year, so there were plenty of best wishes :slightly_smiling_face: 
 - @joostdecock went over the official agenda part, including:
   - The newsletter situation: Explaining that our initial idea was to start a newsletter in 2021, and we sort of assumed that would mean we would send out the first issue at the start of 2021. However, since we haven't yet published a newsletter registration form, that's sort of pointless since we don't have subscribers to send it to
   - The newsletter registration system is part of the v2.11 rollout, which has a deadline of yesterday, so will hopefully be deployed today :see_no_evil: 
   - The issue to rename all master branches to main is still open, but only because we have yet to deploy an update to the website since the changes, and thus we keep it open in case there's any unforeseen issues. However, the work is done and we expect no problems.
- The v2.11 milestone is due. Typically a bump in the minor version means a new pattern. In this case, there is no new pattern, be we felt a minor version was justified because we made significant changes to some of the underlying (frontend) components to implement the new website UI, so this is going to be a rather disruptive change from that point of view.
- #770 was created to investigate fit issues in titan/paco. Since that's our foundational block for future trouser designs, we want to take out some time to look into these in detail, and figure out what's going wrong since there do not seem to be any obvious errors in the measurements, yet some users are not getting good fit results. All help is welcome here
- Brief summary of 2020 and how it was a good year for FreeSewing, we'll publish a blog post on that as part of the 2.11 release
- Then there was some discussion and the new faces got a moment to introduce themselves :slightly_smiling_face: 
- Leo asked what exactly a *body ambassador* was. Joost explained that the idea is that a person who has certain fit challenges (because they are very tall or short, or shaped like this or that) would represent their peers and advocate on their behalf, making sure we take their specific needs into account in their designs
- Karen asked whether she could help with illustrations, and we talked about some of the more intricate features of the SVG format
- Joost reminded everyone that we have an open position to fill: #739 

## Todo

- Add an FAQ on breasts/no breasts and link the getting started guide to it (already done)
- Roll out v2.11 asap: See [the v2.11 milestone](https://github.com/freesewing/freesewing/milestone/12)
- Write the blog post to go along with the 2.11 release: #775
- Let @tangerineshark know about missing illustrations. See [this issue](https://github.com/freesewing/freesewing.org/issues/190)

**Comment by joostdecock** [comment]
Closing in favor of #774 although your comments are still welcome :slightly_smiling_face:

**Comment by tangerineshark** [comment]
I made some illustrations for Teagan as a little experiment (I figured that might be a better place to start than the linked issue above, because Teagan illustrations just need internal consistency and _I think_ no dark mode version). I tried out two different style options - a more basic (a) version and a (b) version that's closer to other illustrations on FS, neither with the maligned polka dots. (The numbers are just which instruction step they're for.) Would either of these (perhaps with modifications) suit a set of illustrations for Teagan's instructions? I can def post this elsewhere if preferred.
![teagan-options](https://user-images.githubusercontent.com/70777269/104823502-15006300-5810-11eb-9cbd-77d66e14e2a7.png)
